code/modules/chemistry/Chemistry-Tools.dm

```diff
@@ -710,15 +710,17 @@ proc/ui_describe_reagents(atom/A)
 	desc = "A set of glass tubes useful for seperating reactants from products. Can be hooked up to many types of containers."
 	icon = 'icons/obj/chemical.dmi'
 	icon_state = "condenser"
-	var/fluid_prefix = "condenser"
 	amount_per_transfer_from_this = 10
 	incompatible_with_chem_dispensers = TRUE //could maybe be ok? idk
 	can_recycle = FALSE //made of glass, but would be a waste and almost certainly accidental so no
 	splash_all_contents = FALSE
 	object_flags = FPRINT | OPENCONTAINER | SUPPRESSATTACK
 	initial_volume = 100
 	accepts_lid = TRUE
+	//prefix for fluid icon state
+	var/fluid_prefix = "condenser"
 	var/list/connected_containers = list() //! the containers currently connected to the condenser
+	var/list/unique_containers = list() //! connected_containers, but without duplicate entries
```

Comment:
copying what I said from discord: what's with the unique_containers var? Surely you can just check if it's in connected_containers and achieve the same thing?